lbz/misc.py

```diff
@@ -131,7 +130,7 @@ def deep_update(dict_to_update: dict, update_data: dict) -> None:
 
 
 def is_in_debug_mode() -> bool:
-    return environ.get("LBZ_DEBUG_MODE", "").lower() == "true"
+    return cast(bool, LBZ_DEBUG_MODE.value)
```

Comment:
If we need to cast the type of `.value`, this means our `ConfigValue` is wrongly implemented 😞

Reply:
Or the parser is not specific enough 🤔